.swiftlint.yml

```diff
@@ -92,16 +97,23 @@ opt_in_rules:
   - test_case_accessibility
   - toggle_bool
   - trailing_closure
+  - unavailable_condition
   - unavailable_function
   - unneeded_parentheses_in_closure_argument
   - unowned_variable_capture
   - untyped_error_in_catch
-  - unused_declaration
-  - unused_import
   - vertical_parameter_alignment_on_call
   - vertical_whitespace_between_cases
+  - void_function_in_ternary
+  - weak_delegate
```

Comment:
As per the SwiftLint changelog: This used to be on by default, but is not anymore.